aiohttp/pytest_plugin.py

```diff
@@ -304,9 +304,13 @@ def aiohttp_server(loop: asyncio.AbstractEventLoop) -> Iterator[AiohttpServer]:
     servers = []
 
     async def go(
-        app: Application, *, port: Optional[int] = None, **kwargs: Any
+        app: Application,
+        *,
+        host: Optional[str] = "127.0.0.1",
```

Comment:
See the type error. host can't be None:
```suggestion
        host: str = "127.0.0.1",
```